core/app/c/[communitySlug]/pubs/[pubId]/edit/page.tsx

```diff
@@ -130,7 +130,7 @@ export default async function Page(props: {
 			}
 		>
 			<div className="flex justify-center py-10">
-				<div className="prose max-w-prose flex-1">
```

Comment:
this originally came from https://github.com/pubpub/platform/pull/980 but it seems like just the `flex-1` is enough to keep the width. adding `prose` changes a bunch of styles, including making `<hr/>`s have huge margins

![image](https://github.com/user-attachments/assets/00f16bba-a90f-4c17-8671-066821270986)

`prose` is used for markdown, but I believe the various markdown places already have `prose` on them, i.e. `StructuralElement`
https://github.com/pubpub/platform/blob/0711707979fe1bc51874d120e33d6626b3f7be32/core/app/components/FormBuilder/FormElement.tsx#L167